src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,60 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 100;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        userNotNullVerify(user);
+        loginVerify(user);
+        ageVerify(user);
+        passwordVerify(user);
+        return storageDao.add(user);
+    }
+
+    private static void userNotNullVerify(User user) {
```

Comment:
move verity to the beginning of the method's name, the same for other methods, name of method should start with verb

Reply:
fixed